web/components/FcSummaryStudyRequest.vue

```diff
@@ -1,127 +1,167 @@
 <template>
-  <section>
-    <v-row class="mt-1 mb-6">
-      <v-col cols="6">
-        <div class="subtitle-1">Status</div>
-        <div class="mt-1 display-1">
-          <div class="align-center d-flex">
-            <v-icon
-              :color="studyRequest.status.color"
-              left>mdi-circle-medium</v-icon>
-            <span>
-              {{studyRequest.status.text}}
+  <div class="pl-5">
+    <section class="pr-5">
+      <v-row class="mt-1 mb-2">
+        <v-col class="pb-0" cols="12">
+          <div class="subtitle-1">Status</div>
+          <FcStatusStudyRequest
+            class="mt-2"
+            :study-request="studyRequest" />
+        </v-col>
+        <v-col cols="6">
+          <div class="subtitle-1">Requester</div>
+          <div class="mt-1 display-1">
+            <span
+              v-if="studyRequest.userId === auth.user.id">
+              {{auth.user.uniqueName}}
+            </span>
+            <span
+              v-else-if="studyRequestUsers.has(studyRequest.userId)">
```

Comment:
can we just add `auth.user` to `studyRequestUsers` in `FcDrawerViewRequest`?  this would avoid having to copy-paste this branching template everywhere we need a username.